lua/nvim-treesitter/parsers.lua

```diff
@@ -1763,6 +1763,15 @@ return {
     maintainers = { '@stsewd', '@theHamsta' },
     tier = 2,
   },
+  qf = {
+    install_info = {
+      revision = 'v1.2.0',
```

Comment:
should be latest commit on the main branch then

(the point of tier 2 is to make it easier to iterate so you don't have to make a release for every change)